build/config/config1.json

```diff
@@ -1,9 +1,11 @@
 {
   "root_dir": "/go/src/github.com/pocket-network",
   "genesis": "build/config/genesis.json",
-  "private_key": "2b000000000000000000000000000000000000000000000000000000000000001ba66c6751506850ae0787244c69476b6d45fb857a914a5a0445a24253f7b810",
+  "private_key": "2e00000000000000000000000000000000000000000000000000000000000000264a0707979e0d6691f74b055429b5f318d39c2883bb509310b67424252e9ef2",
   "pre2p": {
-    "consensus_port": 8080
+    "consensus_port": 8080,
+    "use_raintree": true,
+    "connection_type": "tcp"
```

Comment:
I don't think we should open this for configuration. I think we should stick to TCP for everything for the moment until and do UDP for redundancy, clean-up and churn. So really no point opening this up for config.

Reply:
I'm using it for testing:

```
			Pre2P: &config.Pre2PConfig{
				ConsensusPort:  8080,
				UseRainTree:    true,
				ConnectionType: config.EmptyConnection,
			},
```

This way we inject the fact that we don't need TCP or UDP for mocking purposes.